MainApp/src/main/res/layout/touchpad_container.xml

```diff
@@ -38,9 +38,7 @@
                 android:textColor="#FFFFFF"
                 android:onClick="clickDial"
                 android:text="@string/dial_button"
-                android:paddingStart="40dp"
-                android:paddingEnd="40dp"
```

Comment:
Looks like these are something to do with making LTR/RTL text look right… presumably we can get away without them.

https://stackoverflow.com/questions/12160105/what-are-paddingstart-and-paddingstop